python-checks/src/main/java/org/sonar/python/checks/NoSonarCommentCheck.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.checks;
+
+import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.PythonSubscriptionCheck;
+import org.sonar.plugins.python.api.tree.Token;
+import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.plugins.python.api.tree.Trivia;
+
+import static org.sonar.python.metrics.FileLinesVisitor.containsNoSonarComment;
+
+
+@Rule(key = "NoSonar")
+
+public class NoSonarCommentCheck extends PythonSubscriptionCheck {
+
+  private static final String MESSAGE = "Is //NOSONAR used to exclude false-positive or to hide real quality flaw?";
```

Comment:
```suggestion
  private static final String MESSAGE = "Is #NOSONAR used to exclude false-positive or to hide real quality flaw?";
```
`//` would not be the correct comment opener in Python